pynicotine/gtkgui/ui/settings/chats.ui

```diff
@@ -164,6 +164,7 @@
                         <property name="hexpand">True</property>
                         <property name="label" translatable="yes">Minimum characters required to display drop-down:</property>
                         <property name="mnemonic-widget">min_chars_dropdown_spinner</property>
+                        <property name="sensitive" bind-source="enable_completion_dropdown_toggle" bind-property="active" bind-flags="bidirectional|sync-create"/>
```

Comment:
Add this property to the parent box instead.

Reply:
It is another issue, there should be many multiple matches displayed. The problem is long-standing, I assume it must be something to do with X11 or suchlike, because it the vertical positioning is jumpy in a visually glitchy way.